packages/medusa/src/types/global.ts

```diff
@@ -33,14 +33,26 @@ export type Logger = _Logger & {
   warn: (msg: string) => void
 }
 
+export type sslInterface =
+{
+  ca:string|undefined
+  rejectUnauthorized:boolean|undefined
+}
+
 export type ConfigModule = {
   projectConfig: {
     redis_url?: string
 
     jwt_secret?: string
     cookie_secret?: string
+    database_host?:string|undefined
+    database_port?:number|undefined
+    database_ssl?:sslInterface|undefined,
+    database_username?:string|undefined,
+    database_password?: string | (() => string) | (() => Promise<string>),
+    
 
-    database_url?: string
+    database_url?: string|undefined
```

Comment:
suggestion: You can remove `undefined` since this is done by `?`

Reply:
done